target/linux/lantiq/dts/EASY220.dtsi

```diff
@@ -0,0 +1,295 @@
+#include "vr9.dtsi"
+#include <dt-bindings/input/input.h>
+
+/ {
+
+	compatible = "lantiq,xway", "lantiq,vr9", "lantiq,xrx220";
+
+	chosen {
+		bootargs = "console=ttyLTQ0,115200";
+	};
+
+	aliases {
+		led-boot = &power_g;
+		led-failsafe = &power_a;
+		led-running = &power_g;
+
+		led-dsl = &dsl;
+		led-internet = &lan_g;
+	};
+
+	memory@0 {
+		reg = <0x0 0x3f00000>;
+	};
+
+	ltq_swreset {
+		compatible = "lantiq,ltq_swreset";
+		swreset_pin = <&gpio 7 GPIO_ACTIVE_HIGH>;
+		swreset_bit = <1>;
+		status = "ok";
+	};
+
+	gphy-xrx200 {
+		compatible = "lantiq,phy-xrx200";
+		firmware = "lantiq/vr9_phy22f_a2x.bin";
+		phys = [ 00 01 ];
+	};
+
+//	gpio-keys-polled {
+//		compatible = "gpio-keys-polled";
+//		#address-cells = <1>;
+//		#size-cells = <0>;
+//		poll-interval = <100>;
+/*		reset {
+			label = "reset";
+			gpios = <&gpio 7 GPIO_ACTIVE_HIGH>;
+			linux,code = <0x198>;
+		};*/
+//	};
+
+	gpio-leds {
+		compatible = "gpio-leds";
+
+		power_a: power_amber {
+			label = "power_amber";
+			gpios = <&gpio 0 GPIO_ACTIVE_HIGH>;
+		};
+		power_g: power_green {
+			label = "power_green";
+			gpios = <&gpio 1 GPIO_ACTIVE_LOW>;
+		};
+		lan_amber {
+			label = "lan_amber";
+			gpios = <&gpio 33 GPIO_ACTIVE_LOW>;
+		};
+		lan_g: lan_green {
+			label = "lan_green";
+			gpios = <&gpio 11 GPIO_ACTIVE_LOW>;
+		};
+		dsl_amber {
+			label = "dsl_amber";
+			gpios = <&gpio 8 GPIO_ACTIVE_LOW>;
+		};
+		dsl: dsl_green {
+			label = "dsl_green";
+			gpios = <&gpio 36 GPIO_ACTIVE_LOW>;
+		};
+	};
+};
+
+&gpio {
+	pinctrl-names = "default";
+	pinctrl-0 = <&state_default>;
+
+	state_default: pinmux {
+		mdio {
+			lantiq,groups = "mdio";
+			lantiq,function = "mdio";
+		};
+		phy-rst {
+			lantiq,pins = "io42";
+			lantiq,pull = <0>;
+			lantiq,open-drain = <0>;
+			lantiq,output = <1>;
+		};
+		exin {
+			lantiq,groups = "exin3";
+			lantiq,function = "exin";
+		};
+		/*stp {
+			lantiq,groups = "stp";
+			lantiq,function = "stp";
+		};*/
+		/*nand {
+			lantiq,groups = "nand cle", "nand ale",
+					"nand rd", "nand rdy";
+			lantiq,function = "ebu";
+		};*/
+		/*pci {
+			lantiq,groups = "gnt1", "req1";
+			lantiq,function = "pci";
+		};*/
+		/*led {
+			lantiq,groups = "led";
+			lantiq,function = "led";
+		};*/
+		conf_in {
+			lantiq,pins = "io39"; /* exin3 */
+			lantiq,pull = <2>;
+		};
+		conf_out {
+			lantiq,pins =  // "io21", "io38", /* pcie */
+//					"io12", "io15", /* annexa, annexb */
```

Comment:
Please drop the comments. But this one is rather interesting. Can the same board be used on annex a and annex b lines? Is annex a/b compatibility enabled by pulling these gpios?